src/main/java/teammates/ui/controller/InstructorFeedbackAbstractAction.java

```diff
@@ -0,0 +1,170 @@
+package teammates.ui.controller;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import teammates.common.datatransfer.FeedbackSessionType;
+import teammates.common.datatransfer.attributes.CourseAttributes;
+import teammates.common.datatransfer.attributes.FeedbackSessionAttributes;
+import teammates.common.datatransfer.attributes.InstructorAttributes;
+import teammates.common.util.Const;
+import teammates.common.util.EmailType;
+import teammates.common.util.SanitizationHelper;
+import teammates.common.util.TimeHelper;
+
+public abstract class InstructorFeedbackAbstractAction extends Action {
+
+    protected FeedbackSessionAttributes extractFeedbackSessionDataHelper(
+            FeedbackSessionAttributes newSession, List<String> sendRemainderEmailsList) {
+        return this.extractFeedbackSessionDataHelper(newSession, sendRemainderEmailsList);
+    }
+
+    protected FeedbackSessionAttributes extractFeedbackSessionData() {
+        // TODO assert parameters are not null then update test
+        // TODO make this method stateless
+
+        FeedbackSessionAttributes newSession = new FeedbackSessionAttributes();
+        newSession.setCourseId(getRequestParamValue(Const.ParamsNames.COURSE_ID));
+        newSession.setFeedbackSessionName(SanitizationHelper.sanitizeTitle(
+                getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_NAME)));
+        newSession.setFeedbackSessionType(FeedbackSessionType.STANDARD);
+        newSession = setTime(newSession);
```

Comment:
I don't think you need to return `newSession` each time you call a `set*` method.